cnv/cnv_release_notes/cnv-release-notes.adoc

```diff
@@ -0,0 +1,70 @@
+[id="cnv-release-notes"]
+= {RN_BookName}
+include::modules/cnv-document-attributes.adoc[]
+:context: cnv-release-notes
+toc::[]
+
+
+== Product overview
+
+include::modules/cnv-introduction-to-cnv.adoc[leveloffset=+2]
+
+== New and changed features
+
+=== Supported binding methods
+* Open vSwitch (OVS) is no longer recommended and should not be used
+in {ProductName} 2.0.
+* For the default Pod network, `masquerade` is the only recommended binding
+method. It is not supported for non-default networks.
+* For secondary networks, use the `bridge` binding method.
+
+=== UI improvements
```

Comment:
is it ok to use acronym here, hm?

Reply:
I changed it to "Web console improvements." Thanks, @alexxa